pyface/timer/do_later.py

```diff
@@ -55,7 +55,7 @@ def do_after(interval, callable, *args, **kwargs):
         The time interval in milliseconds to wait before calling.
     callable : callable
         The callable to call.
-    *args, **kwargs : tuple, dict
+    *args, \**kwargs : tuple, dict
```

Comment:
While this got rid of the warning, the rendered parameters still look odd (but they were already looking odd before this PR).
![Screenshot 2020-12-15 at 17 44 51](https://user-images.githubusercontent.com/3673984/102251902-504fff00-3efd-11eb-9a03-a34a707ba254.png)

Looks like we need to split `*args` and `**kwargs` into its own line... something like

```suggestion
    \*args
        Positional arguments to be passed through to the callable.
    \*\*kwargs
        Keyword arguments to be passed through to the callable.
```